yahoofantasy/resources/league.py

```diff
@@ -28,6 +28,21 @@ def teams(self, persist_ttl=DEFAULT_TTL):
             teams.append(t)
         return teams
 
+    def players(self, persist_ttl=DEFAULT_TTL):
+        logger.debug("Looking up players")
+        START = 0
+        COUNT = 25
```

Comment:
these START and COUNT variables may come from a constants/config file. As there are no other usages, I choose to use them inside the method directly.

25 is the max limit on yahoo fantasy API. I tried to use this max value to decrease the number of requests. I'm not sure is there any benefit for the user to use another value on count.